### DISCUSSION TITLE
Add a setting to disable the screenshot capture before the new ticket url opens

### DISCUSSION THREAD

**Discussion by DeltinoRosso** [glpi-project/glpi-agentmonitor#16] [Ideas]
Good Day Everyone, 
I really appreciate your work and thank you for the effort that you put in maintaining this software. 

I would like to ask for a feature that adds a a setting so that would be possible to disable the screenshot that is taken before the opening of new url ticket. 

Thank you very much and best regards

**Comment by redddcyclone** [comment]
Good day @DeltinoRosso

Thanks a lot for your words

Ok, I'll be implementing this feature soon.

@g-bougard, could you add another setting on the installer parameters for this? I plan to use ```NewTicket-Screenshot``` as the registry value. Would ```AGENTMONITOR_NEWTICKET_SCREENSHOT``` be a suitable param?

**Reply by g-bougard** [reply]
Hi @redddcyclone 

okay, not a problem.

What should I set as default value during installation ? And can you confirm to set it as string ?

**Reply by redddcyclone** [reply]
@g-bougard 

The default should be ```1```. I have set it as REG_DWORD on code... must I change it to string (REG_SZ)?

**Reply by g-bougard** [reply]
No, no need, I'll take care to create it as REG_DWORD too with this default value.

**Reply by g-bougard** [reply]
Just push the update in [4fb7363](https://github.com/glpi-project/glpi-agent/commit/4fb7363a05e62e13ae2a27c745e77f13c2071ca0)

You should be able to test it and validate it works as expected using next nightly build.

**Reply by g-bougard** [reply]
This update is only related to `NewTicket-Screenshot` configuration setup from the installer.

I'll include a new GlpiAgent-Monitor version only when missing strings translation are updated or you think we can no more wait after contributors for that.

**Reply by redddcyclone** [reply]
Hi @g-bougard 

OK, thanks a lot as always! I'll wait a bit for the contributors to answer like last time.

**Comment by redddcyclone** [comment]
@DeltinoRosso your suggestion was added in the code and will likely be released in the next Agent release. Thanks a lot for the suggestion!

**Reply by DeltinoRosso** [reply]
Thank you very much for your fast response and for the implementation of the feature.
